### ISSUE TITLE
Portable

### ISSUE THREAD

**Issue by smaragdus** [saenzramiro/rambox#35]
The so called portable versions of Rambox:
"Rambox-win32-ia32-portable.zip" & "Rambox-win32-x64-portable.zip"
are not really portable because they save program data and settings not inside program folder but in AppData (C:\Users\UserName\AppData\Roaming\Rambox)
which means that one cannot use Rambox from external drives on other machines without leaving traces and junk behind.

I would appreciate a truly portable version which stores data and settings in Rambox folder (start parameter, option to make Rambox portable, asking the user where to save settings on first run, etc).

I also noticed that "About" menu (Help-About Rambox) is not working.

**Comment by saenzramiro** [comment]
That's true, is not really portable. Doing a really portable version for now, is not important for me... Maybe in the future or someone can do it and share.

Thanks!

**Comment by iliyang** [comment]
+1 for a portable version!

**Comment by franciscoluiz** [comment]
+1 for a truly portable version!

**Comment by NucsuM** [comment]
+1 for a truly portable version!

**Comment by duarteframos** [comment]
+1 for a truly portable version!

**Comment by starsoccer** [comment]
+1 for portable version

Really the only change that is needed is the ability to select the save location for program data

**Comment by GranitCircle** [comment]
+1 for portable version

**Comment by wuschelconne** [comment]
+1 for a truly portable version!

**Comment by AdKiller** [comment]
+100 for portable version

**Comment by franciscoluiz** [comment]
+1000 for portable version

**Comment by kprkpr** [comment]
+1 for portable version

**Comment by saenzramiro** [comment]
In latest version 0.5.9, portable seems to work ok. Can anyone check this?

**Comment by kprkpr** [comment]
Where is this portable version? I didnt see it, and the "Rambox-0.5.9-x64-win.zip" uses %appdata%\Rambox

**Comment by wokawoka** [comment]
+1 
by the way, thanks for this great application

**Comment by eadmaster** [comment]
A cmdline switch to change the user's data path would solve this.
[btw for Windows i've found this](https://portableapps.com/node/56679).

**Comment by Striler** [comment]
Any news on this important topic?

**Comment by AdKiller** [comment]
It's been 3 years... we are not going to get a portable app. sad...

**Comment by pandruszkow** [comment]
It's even worse now, due to the automatic update feature. It just installs Rambox systemwide any time a new version is released, without asking.

**Comment by crazy-max** [comment]
Hi guys,

Just wanted to let you know that [Rambox portable](https://portapps.io/app/rambox-portable/) is available through [Portapps](https://portapps.io/).

**Comment by sm131313** [comment]
There is a Portable version for Windows. 
Why can't we also have one for MacOS?

**Comment by yardstudio** [comment]
> Hi guys,
> 
> Just wanted to let you know that [Rambox portable](https://portapps.io/app/rambox-portable/) is available through [Portapps](https://portapps.io/).

I like portapps.io, but can community investigate, if it is safe project? :)
